src/main/resources/application.yaml

```diff
@@ -196,4 +196,5 @@ launchdarkly:
   offline-mode: false
   env: ${LAUNCH_DARKLY_ENV:default}
 
-
+cron:
```

Comment:
does this need to be in the charts to set it per env?

Reply:
yeah unfortunately, that's how we're going to configure that it only runs on a single instance

Reply:
is it worth adding these to the chart now?